src/app/store/services/shared.service.ts

```diff
@@ -10,10 +10,12 @@ import { Folder, MailFolderType } from '../models';
 import { AllowIn } from 'ng-keyboard-shortcuts';
 import { GenericFolderComponent } from '../../mail/mail-list/mail-folder/generic-folder/generic-folder.component';
 import { MailComponent } from '../../mail/mail.component';
-import { MailDetailComponent } from '../../mail/mail-detail/mail-detail.component';
+import { PricingPlan } from '../datatypes';
 
 @Injectable()
 export class SharedService {
+  static PRICING_PLANS: any;
+  static PRICING_PLANS_ARRAY: Array<PricingPlan> = [];
```

Comment:
Instead of using static Arrays I would suggest to use `Subject` and if needed create getter methods but keep these fields private. Also, I think its better to create separate service for Pricing Plans.